Recrutify/Recrutify.Services/DTOs/SaveArgsDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Recrutify.Services.DTOs
```

Comment:
please create new folder Events and move here

Reply:
Move here event SaveDetailsHandler UpdateStatusByIdsAsyncComlited;
OnUpdateStatusByIdsAsyncComlited from StatusChangeEventHandler

Reply:
see as example https://www.tutorialsteacher.com/csharp/csharp-event
it should be similar structure